roles/update-pkgs/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+- name: Update packages and kernel to latest available versions
+  package:
```

Comment:
what will happen when this run on the RHEL os? where is the rhn registration happening?

Reply:
Moved in block that deals with the registration of the OS to the update task..